docs/tutorial/notifications.md

```diff
@@ -101,41 +111,41 @@ handling replies requires the use of
 which helps with registering the required COM components and calling your
 Electron app with the entered user data.
 
-#### Quiet Hours / Presentation Mode
+#### Query notification state
 
 To detect whether or not you're allowed to send a notification, use the
-userland module [electron-notification-state](https://github.com/felixrieseberg/electron-notification-state).
+userland module [`windows-notification-state`][windows-notification-state].
```

Comment:
`electron-notification-state` is a module that @felixrieseberg owns. I think it uses Felix's other `windows-notification-state` and `macos-notification-state` modules under the hood, but its package.json locks those modules to `^1.0.0` when v2 has been released for both.

I've changed this recommendation around to use the platform-specific modules instead.